nss-day-cohort-46/rare-rest-rare-api-cool-cake
# Cr posts search by tag

# Description

User can search posts by tag.

Closes #44

## Type of change



# Testing Instructions

Make sure you have enough posts, tags, posttags.

`git fetch --all`
`git checkout cr-posts-searchByTag`
`python3 manage.py loaddata tags`
`python3 manage.py loaddata posts`
`python3 manage.py loaddata posttags`
`python3 manage.py runserver`

For client-side instructions, see frontend pull request.

# For Postman

Logged in as staff, when searching posts, staff user will see all posts regardless of approval status and/or publication date.
Logged in as non-staff, when searching posts, user will see all approved posts that do not have a future publication date.

Make a `GET` request with one of the words below to `http://localhost:8000/posts/filterByTag?tag={word goes here}`

`
strategy
sports
adventure
battle
dice
battle
`

# Checklist:
